Deployed Stoa doesn't start

```
Apr 02 03:59:37 eu-002 systemd[1]: Started Stoa: API server for Agora node.
Apr 02 03:59:40 eu-002 docker[2185315]:
Apr 02 03:59:40 eu-002 docker[2185315]: > stoa@0.0.0 start /stoa/bin
Apr 02 03:59:40 eu-002 docker[2185315]: > NODE_ENV=production ts-node src/main.ts "-c" "/stoa/wd/config.yaml"
Apr 02 03:59:40 eu-002 docker[2185315]:
Apr 02 03:59:44 eu-002 docker[2185315]: Cannot read property 'genesis_timestamp' of undefined
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! code ELIFECYCLE
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! errno 1
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! stoa@0.0.0 start: `NODE_ENV=production ts-node src/main.ts "-c" "/st>
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! Exit status 1
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR!
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! Failed at the stoa@0.0.0 start script.
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! This is probably not a problem with npm. There is likely additional >
Apr 02 03:59:44 eu-002 docker[2185315]:
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR! A complete log of this run can be found in:
Apr 02 03:59:44 eu-002 docker[2185315]: npm ERR!     /root/.npm/_logs/2021-04-02T03_59_44_752Z-debug.log
Apr 02 03:59:46 eu-002 docker[2185315]:
Apr 02 03:59:46 eu-002 systemd[1]: stoa.service: Main process exited, code=exited, status=1/FAILURE
Apr 02 03:59:46 eu-002 systemd[1]: stoa.service: Failed with result 'exit-code'.
Apr 02 03:59:46 eu-002 systemd[1]: stoa.service: Scheduled restart job, restart counter is at 5.
Apr 02 03:59:46 eu-002 systemd[1]: Stopped Stoa: API server for Agora node.
Apr 02 03:59:46 eu-002 systemd[1]: stoa.service: Start request repeated too quickly.
Apr 02 03:59:46 eu-002 systemd[1]: stoa.service: Failed with result 'exit-code'.
Apr 02 03:59:46 eu-002 systemd[1]: Failed to start Stoa: API server for Agora node.
```

Most likely related to https://github.com/bosagora/stoa/commit/cb2213bbc8b27c2395ca41ddc523a57a8c1fbd29

--- a/src/modules/common/Config.ts
+++ b/src/modules/common/Config.ts
@@ -305,7 +305,7 @@ export class ConsensusConfig implements IConsensusConfig
      * Constructor
      * @param genesis_timestamp The genesis timestamp
      */
-    constructor (genesis_timestamp?: number)
+    constructor ()
     {
         const defaults = ConsensusConfig.defaultValue();
         this.genesis_timestamp = defaults.genesis_timestamp;
@@ -317,8 +317,9 @@ export class ConsensusConfig implements IConsensusConfig
      */
     public readFromObject (config: IConsensusConfig)
     {
-        if (config.genesis_timestamp)
-            this.genesis_timestamp = config.genesis_timestamp;
+        let conf = extend(true, {}, ConsensusConfig.defaultValue());
+        extend(true, conf, config);
+        this.genesis_timestamp = conf.genesis_timestamp;
     }
 
     /**

